### PR TITLE
Remove requirement for conan

### PR DESCRIPTION
**Pull request by soehrl**
I removed the requirements for conan, updated the readme file and fixed alot of warnings for gcc.

I am currently not able to test if WebRTC is working. However, as changed nothing in that regard (because WebRTC was never added through conan) it should continue to work.

### CODE REVIEW

File: cmake/suppress_warnings.hpp.in [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//------------------------------------------------------------------------------
+
+#ifndef CONTRA_SUPPRESS_WARNINGS_
+#define CONTRA_SUPPRESS_WARNINGS_
```
Comment (ptoharia) [inline cmake/suppress_warnings.hpp.in]: @soehrl why contra here?

**Comment by soehrl** [comment]
@NadirRoGue Oh that's true.. I somehow was under the impression that there aren't any. I'll adjust the readme

File: cmake/suppress_warnings.hpp.in (hunk 1 above)
Comment (soehrl) [reply cmake/suppress_warnings.hpp.in]: Oh, that's just a copy'n'paste error

File: README.md [hunk 2]
```diff
@@ -3,20 +3,44 @@
...
+sudo apt install libx264-dev
+sudo apt install libswscale-dev
+sudo apt install libpng-dev
+```
 
-* *CMake* >= 3.1 (https://cmake.org/)
-* *Conan*: `pip install conan`
+Unfortunately, there is no package for poco, you have to build it from source using the following commands:
+```
+git clone https://github.com/pocoproject/poco.git
```
Comment (ptoharia) [inline README.md]: @soehrl in 18.04 I have installed package libpoco-dev (version 1.8.0.1) and webstreamer builds

Comment (soehrl) [reply README.md]: @ptoharia Fixed that already, see edd59a6

**Comment by juanpebm** [comment]
Thanks for the update Simon. I haven't tested WebRTC config but without it, webstreamer runs ok with the new cmake configuration. Only one thing, we need to add the building directory of webstreamer to our cmake to include some files generated in configuration time (for example export.hpp). This could be solved using installation rules in cmake, but at this moment is ok for me.

**Comment by soehrl** [comment]
@juanpebm No worries :) Yes, we should definitely have an install target that manages this stuff. I added it recently to the in-situ-pipeline and will integrate it also into webstreamer. Expect another PR in the next few days!

**Pull request merged**